templates/default/parallelcluster-ebsnvme-id.erb

```diff
@@ -1,28 +1,24 @@
-#!/usr/bin/env python
+#!<%= node['cfncluster']['cookbook_virtualenv_path'] %>/bin/python
```

Comment:
why convert python to vitual environment? Doesn't python at system level use same version as python at virtualenv ?